<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysTrayIconConfigWidget</class>
 <widget class="QWidget" name="SysTrayIconConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkSysTrayOn">
     <property name="text">
      <string>Dock into System Tray</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkRestoreOnHover">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Restore Windows upon mouse hover</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkReplaceQuitButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Replace main window's quit button by an iconify and quit button bar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkMinToSysTray">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Minimize button sends the window to system tray</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chkSysTrayOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkRestoreOnHover</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSysTrayOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkReplaceQuitButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSysTrayOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkMinToSysTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
